src/winconpty/device.h

```diff
@@ -0,0 +1,25 @@
+/*++
+Copyright (c) Microsoft Corporation
+Licensed under the MIT license.
+
+Module Name:
+- device.h
+
+Abstract:
+- This header exists to reduce the differences in winconpty

```

Comment:
Well yes. And it also exists to go do the ol' reach to somewhere else to find NtOpenFile since you need that namespace. Leave such a comment since it's not obvious except to seasoned observers?